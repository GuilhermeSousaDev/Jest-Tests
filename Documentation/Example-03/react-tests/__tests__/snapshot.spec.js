describe('Snapshot', () => {
    /*test('renderer correctly', () => {
        const three = renderer.create(
            <a href="http://www.instagram.com">Instagram</a>
        ).toJSON()

        expect(three).toMatchSnapshot()
    }); */

    it('will fail every time', () => {
        const user = {
          createdAt: new Date(),
          id: Math.floor(Math.random() * 20),
          name: 'LeBron James',
        };
      
        expect(user).toMatchSnapshot();
      });
});