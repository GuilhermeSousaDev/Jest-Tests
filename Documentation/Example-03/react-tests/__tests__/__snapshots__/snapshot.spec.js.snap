// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot will fail every time 1`] = `
Object {
  "createdAt": 2022-06-09T10:27:12.481Z,
  "id": 0,
  "name": "LeBron James",
}
`;
